### PR TITLE
 update Company Details error messages to be more in line with Design System examples 

### PR DESCRIPTION
**Pull request by gidsg**
https://trello.com/c/AnpNWBC7/893-2-update-error-messages-for-confirm-company-details-journey
Re-word error messages to be closer to those used in Design System examples from https://design-system.service.gov.uk/components/error-message/

@NoraGDS please may you review the wording changes in `app/main/forms/suppliers.py`?

### CODE REVIEW

File: tests/app/main/test_suppliers.py [hunk 1]
```diff
@@ -2816,32 +2811,32 @@ def test_add_registration_number_page_loads(self):
...
+                'Enter a Companies House number'
             ),
             (
                 {'has_companies_house_number': 'Yes',
                  'companies_house_number': '123456789',
                  'other_company_registration_number': ''
                  },
-                'Enter your 8 digit Companies House number.',
-                'Enter your 8 digit Companies House number.'
+                "Companies House number must be 8 characters",
```
Comment (NoraGDS) [inline tests/app/main/test_suppliers.py:2830]:
```suggestion
                "Your Companies House number must be 8 characters",
```

File: tests/app/main/test_suppliers.py [hunk 2]
```diff
@@ -2816,32 +2811,32 @@ def test_add_registration_number_page_loads(self):
...
+                "Companies House number must be 8 characters"
```
Comment (NoraGDS) [inline tests/app/main/test_suppliers.py:2831]:
```suggestion
                "Your Companies House number must be 8 characters"
```

File: app/main/forms/suppliers.py [hunk 3]
```diff
@@ -102,7 +102,7 @@ class AddCompanyRegistrationNumberForm(FlaskForm):
         validators=[
             Optional(),
             Regexp(r'^([0-9]{2}|[A-Za-z]{2})[0-9]{6}$',
-                   message="Enter your 8 digit Companies House number."
+                   message="Companies House number must be 8 characters"
```
Comment (NoraGDS) [inline app/main/forms/suppliers.py:105]:
```suggestion
                   message="Your Companies House number must be 8 characters"
```

File: app/main/forms/suppliers.py [hunk 4]
```diff
@@ -133,54 +133,54 @@ def validate(self):
...
+        Length(max=20, message="Phone number must be %(max)d characters or fewer")
     ])
 
 
 class DunsNumberForm(FlaskForm):
     duns_number = DMStripWhitespaceStringField('DUNS Number', validators=[
-        InputRequired(message="Enter your 9 digit DUNS number."),
-        Regexp(r'^\d{9}$', message="Enter your 9 digit DUNS number."),
+        InputRequired(message="Enter your 9 digit DUNS number"),
+        Regexp(r'^\d{9}$', message="DUNS number must be 9 digits"),
```
Comment (NoraGDS) [inline app/main/forms/suppliers.py:168]:
```suggestion
        Regexp(r'^\d{9}$', message="Your DUNS number must be 9 digits"),
```

**Comment by gidsg** [comment]
@NoraGDS I made the suggested changes in 58039c1, please may you review and mark the PR as Approved if you are happy with them?

**Review by NoraGDS — APPROVED** [review]

**Pull request merged**